packages/web-components/package.json

```diff
@@ -90,9 +90,9 @@
     "webpack": "^4.43.0"
   },
   "dependencies": {
-    "@microsoft/fast-colors": "^5.1.0",
+    "@microsoft/fast-colors": "^5.1.3",
     "@microsoft/fast-element": "^1.5.0",
-    "@microsoft/fast-foundation": "^2.8.1",
+    "@microsoft/fast-foundation": "^2.8.2",
```

Comment:
I *think* this is at 2.10 now: https://www.npmjs.com/package/@microsoft/fast-foundation